web/client/plugins/featuregrid/FeatureEditor.jsx

```diff
@@ -292,11 +295,12 @@ const EditorPlugin = compose(
         // due to multiple renders of plugins in contexts (one with default props, then with context props)
         // the options have to be updated when change.
         componentDidUpdate(oldProps) {
-            const newOptions = pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
-            const oldOptions = pick(oldProps, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
+            const newOptions = pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions', 'isSyncWmsActive']);
+            const oldOptions = pick(oldProps, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions', 'isSyncWmsActive']);
             if (!isEqual(newOptions, oldOptions) ) {
                 this.props.onMount(newOptions);
             }
+            if (newOptions.isSyncWmsActive) return;
```

Comment:
please avoid inline if

also why this condition here? you are going to skip the following part, but maybe is intended?
i think this part is the one raising the issue describe in the review comment